client/src/main/kotlin/io/spine/examples/chatspn/desktop/LoginPage.kt

```diff
@@ -56,7 +59,13 @@ public fun LoginPage(
                 emailErrorText.value = "Email field must not be empty"
             }
             if (!emailErrorState.value) {
-                toChats()
+                try {
+                    client.logIn(emailState.value)
```

Comment:
The problem here and in a similar place below is that we modify things in two-way synchronous fashion.

Compose ideology is different. The actions triggered from UI should lead to seamless updates of the UI model, so that UI model re-renders itself afterwards. Asynchronously.

I would do the following:

1) Have a concept of current UI state somewhere in the UI model. Its modifications should be asynchronous. For example, upon a domain event (to which we subscribe), or an update of projection.
2) Instead of catching the exception synchronously, we should subscribe to the related signal, and update the UI model.

Let's discuss vocally, if needed.